planetwars/client/app/lib/match/index.ts

```diff
@@ -1,7 +1,3 @@
-import { MatchLog } from '.';
-
 export { MatchLog, PlayerMap } from './MatchLog';
 export * from './types';
 export * from './utils';
-// TODO: this is not exactly ideal
-export { PwTypes } from 'mozaic-client';
```

Comment:
What happened with these btw?

Reply:
They were not exactly ideal

Reply:
Nothing happened, they are just not included in this file anymore :)